src/test/java/hudson/plugin/versioncolumn/VersionMonitorLoggerTest.java

```diff
@@ -0,0 +1,85 @@
+package hudson.plugin.versioncolumn;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.*;
+
+import hudson.model.Computer;
+import hudson.remoting.VirtualChannel;
+import java.io.IOException;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import java.util.logging.Logger;
+import jenkins.security.MasterToSlaveCallable;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+
+class VersionMonitorLoggerTest {
+
+    private VersionMonitor.DescriptorImpl descriptor;
+    private Computer computer;
+    private VirtualChannel channel;
+    private Logger logger;
+    private TestLogHandler handler;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
I think that the Jenkins test harness provides [LoggerRule](https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/LoggerRule.html) for cases where log recording can help a test.  Could you investigate if it can be used here?